src/AnnotationDialog.js

```diff
@@ -329,17 +328,11 @@ class AnnotationDialog extends EventEmitter {
      * @return {void}
      * @protected
      */
-    addSortedAnnotations(annotations) {
+    sortAnnotationsList(annotations) {
         // Sort annotations by date created
-        const sorted = Object.keys(annotations).map((key) => annotations[key]);
-        sorted.sort((a, b) => {
-            return new Date(a.created) - new Date(b.created);
-        });
-
-        // Add sorted annotations to dialog
-        sorted.forEach((annotation) => {
-            this.addAnnotationElement(annotation);
-        });
+        this.annotations = Object.keys(annotations).map((key) => annotations[key]);
+        this.annotations.sort((a, b) => new Date(a.created) - new Date(b.created));
```

Comment:
Are the date objects needed or can the integers be compared directly?

Reply:
Dates are provided as a string format from the API such as "2016-10-30T14:19:56" so we can't compare directly :-/